static/app/utils/metrics/dashboardImport.tsx

```diff
@@ -315,9 +315,11 @@ export class WidgetParser {
     const appliedFunctionMatch = str.match(/\.(\w+)\(\)/);
     const appliedFunction = appliedFunctionMatch ? appliedFunctionMatch[1] : undefined;
 
-    if (!op) {
-      this.errors.push(`widget.request.query - could not parse op: ${str}, assuming sum`);
-      op = 'sum';
+    if (!aggregation) {
+      this.errors.push(
+        `widget.request.query - could not parse aggregation: ${str}, assuming sum`
+      );
```

Comment:
are we displaying these errors somewhere to clients? shouldn't we use the translation function here?

Reply:
yes, but they are visible only to sentry staff since this is an internal feature

Reply:
ok we just have to remember to update it when the feature is available to clients